Userland/Libraries/LibAudio/PulseAudioWrappers.cpp

```diff
@@ -16,6 +16,13 @@ ErrorOr<NonnullRefPtr<PulseAudioContext>> PulseAudioContext::instance()
     // Use a weak pointer to allow the context to be shut down if we stop outputting audio.
     static WeakPtr<PulseAudioContext> the_instance;
     static Threading::Mutex instantiation_mutex;
+    // Lock and unlock the mutex to ensure that the mutex is fully unlocked at application
+    // exit.
+    static ScopeGuard application_exit_guard { []() {
```

Comment:
We could just call `atexit` here instead of creating a global variable to make it extremely explicit that that's the behavior we're looking for.

```
atexit(+[] {
  instantiation_mutex.lock();
  instantiation_mutex.unlock();
});
```

Reply:
Done!